### PR TITLE
fix: fix skip replay env flag

### PR DESCRIPTION
**Pull request by crepererum**

### CODE REVIEW

File: src/commands/run.rs [hunk 1]
```diff
@@ -241,8 +271,8 @@ Possible values (case insensitive):
     pub max_http_request_size: usize,
 
     /// Skip replaying the write buffer and seek to high watermark instead.
-    #[structopt(long = "--skip-replay", env = "IOX_SKIP_REPLAY")]
-    pub skip_replay_and_seek_instead: bool,
+    #[structopt(long = "--skip-replay", env = "IOX_SKIP_REPLAY", default_value = "no")]
```
Comment (NGA-TRAN) [inline src/commands/run.rs:274]: Does it mean the default is the replay turning on (no skip replay)? Do we want to opposite one?

**Review by NGA-TRAN — APPROVED** [review]

File: src/commands/run.rs [hunk 2]
```diff
@@ -26,6 +26,36 @@ pub enum Error {
 
 pub type Result<T, E = Error> = std::result::Result<T, E>;
 
+/// Boolean flag that works with environment variables.
+///
+/// Workaround for https://github.com/TeXitoi/structopt/issues/428
+#[derive(Debug)]
+pub enum YesNo {
```
Comment (tustvold) [inline src/commands/run.rs:33]: BooleanFlag might be a more descriptive name, but :shrug:

**Review by tustvold — APPROVED** [review]
LGTM - bit silly that it is necessary though :laughing:

File: src/commands/run.rs (hunk 1 above)
Comment (tustvold) [reply src/commands/run.rs:274]: Skipping replay is an exceptional scenario, as it implies we will skip writes that have been made durable to the write buffer and acknowledged to clients by the routing tier. I think it should be off by default, as this does

Comment (crepererum) [reply src/commands/run.rs:274]: I think the non-tools default should be "perform replay as designed" aka "do not replay" aka "skip-replay=no". I'll override this env variable for our deployment though.

File: src/commands/run.rs (hunk 2 above)
Comment (crepererum) [reply src/commands/run.rs]: done

**Pull request merged**